### ISSUE TITLE
Accessing jobs in subfolders

### ISSUE THREAD

**Issue by mostealth** [pbelskiy/aiojenkins#21]
job api only handles jobs in the Jenkins server root folder, it does not seem to be possible to list all jobs or access job information (among others) for jobs in subfolders

**Comment by pbelskiy** [comment]
Hello! Thanks for the issue, but could you please tell more about it. How can I reproduce it?

- Are you using `jenkins.jobs.get_all()` and don't see your job? Which is in view? 
- Or you wanna see somehow jobs exactly in some view using it name?
- Or you mean folder within CloudBees Folders Plugin?

**Comment by mostealth** [comment]
Hi Petr

The later. I am using `jenkins.jobs.get_all()` on a jenkins server that has no jobs in the root folder but a folder with jobs inside.
The reply to `jenkins.jobs.get_all()` is something like

```
[{'_class': 'com.cloudbees.hudson.plugins.folder.Folder',
  'name': 'FOLDER1',
  'url': 'https://myserver/job/FOLDER1/'},
  ...
]
```

I cant' see the way to discover the jobs inside FOLDER1.
Keep in mind that FOLDER1 could contain subfolders with subfolders inside.

**Comment by pbelskiy** [comment]
@mostealth 

Hello again, I've fixed it, released under version 0.7.0

https://pypi.org/project/aiojenkins/

If there are still some problems let me now please or I will close this issue.

**Comment by pbelskiy** [comment]
@mostealth 

So, I will close it, reopen if there are still any issues.
